### PR TITLE
RAVTDBQ-450: Support for INTERVAL Datatypes in DDL

### PR DESCRIPTION
**Pull request by Mahesh-Raut**
RAVTDBQ-450: Support for INTERVAL_DAY, INTERVAL_HOUR, INTERVAL_MINUTE, and INTERVAL_MONTH data types in DDL.

### CODE REVIEW

File: core/src/main/java/org/apache/calcite/sql/dialect/BigQuerySqlDialect.java [hunk 1]
```diff
@@ -1927,6 +1927,10 @@ private boolean isIntervalHourAndSecond(SqlCall call) {
       case INTERVAL_DAY_SECOND:
       case INTERVAL_DAY_MINUTE:
       case INTERVAL_DAY_HOUR:
+      case INTERVAL_DAY:
+      case INTERVAL_HOUR:
```
Comment (sauajmera) [inline core/src/main/java/org/apache/calcite/sql/dialect/BigQuerySqlDialect.java:1931]: can we check for INTERVAL_YEAR and INTERVAL_SECOND also
and update the test case accordingly.

Comment (Mahesh-Raut) [reply core/src/main/java/org/apache/calcite/sql/dialect/BigQuerySqlDialect.java:1931]: Done

**Review by sauajmera — APPROVED** [review]
L1 completed.

**Comment by vijayjogi-dm** [comment]
L2 is done

**Pull request merged**